.eslintrc.yml

```diff
@@ -2,6 +2,12 @@ extends: airbnb-base
 plugins:
   - import
 rules:
+  no-underscore-dangle:
+  - error
+  - allowAfterThis: true
+  space-before-function-paren:
+  - error
+  - named: always
```

Comment:
I guess, these are here because of the conflict of eslint and semistandard, right? To be honest I like the 

```javascript
function addProperty(
```
more